libinstall/CentPlugins.sh

```diff
@@ -15,7 +15,13 @@ echo -e "$line"
 
 ###### Check disk space
 check_tmp_disk_space
-[ "$?" -eq 1 ] && purge_centreon_tmp_dir
+if [ "$?" -eq 1 ] ; then
```

Comment:
Please remove the entire CentPlugins.sh. It seems like it is not needed anymore.

Reply:
please check "install.sh +406"